packages/graphile-build-pg/src/plugins/PgIntrospectionPlugin.js

```diff
@@ -252,6 +257,9 @@ export default (async function PgIntrospectionPlugin(
         memo[x[attrKey]][x[attrKey2]] = x;
         return memo;
       }, {});
+
+    //TODO seems like a confusing namespace convention to put id lookups under kind lookups
+    //TODO maybe these should be maps instead of POJO
```

Comment:
I don't see the advantage of moving to Maps here; they are typically slower and IDs are always strings so POJO works well. Also this would be a breaking change, plugins depend on this structure. (One more thing is that this is effectively a cacheable JSON structure currently (which might come in handy); using maps would complicate.)

Reply:
I agree completely. No maps!